services/brig/src/Brig/API.hs

```diff
@@ -180,12 +184,15 @@ sitemap o = do
     post "/i/users/blacklist" (continue addBlacklist) $
         param "email" ||| param "phone"
 
+    get "/i/users/:uid/can-be-deleted/:tid" (continue canBeDeleted) $
```

Comment:
Why do we pass the teamID here?

Reply:
to check whether the team would be orphaned.  brig could ask galley for this, but that would mean two more network roundtrips, and it seemed to @jschaul and me that it is more elegant to provide it explicitly.